atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -197,7 +198,13 @@ protected CassandraKeyValueService(Logger log,
                 configManager.getConfig().poolSize() * configManager.getConfig().servers().size()));
         this.log = log;
         this.configManager = configManager;
-        this.clientPool = new CassandraClientPool(configManager.getConfig());
+        try {
+            this.clientPool = InitializationCheckerProxy.newProxyInstance(
+                    new Class<?>[] {CassandraKeyValueServiceConfig.class}, new Object[] {configManager.getConfig()},
+                    CassandraClientPool.class);
+        } catch (Exception e) {
+            throw com.google.common.base.Throwables.propagate(e);
```

Comment:
i don't think we want to handle exceptions like this, see https://github.com/google/guava/wiki/Why-we-deprecated-Throwables.propagate